main/main.cpp

```diff
@@ -3,8 +3,9 @@
 #include <cerrno>
 #include <fstream>
 #include <iostream>
+#include <sstream>
```

Comment:
Interesting. Not sure why this compiles without this on linux.

Reply:
My best suggestion is that the different C++ standard libraries have different include trees. You might be able to reproduce it by linking against libcxx (via llvm/clang) rather than GCC (and I, vice versa, by trying libstdc++)

Reply:
Yeah, I assume that's it, but weird.